wcfapp/WcfClient/Program.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.ServiceModel;
+
+namespace WcfClient
+{
+    class Program
+    {
+        static string host;
+
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Client OS: {0}", Environment.OSVersion);
+            host = Environment.GetEnvironmentVariable("host") ?? "localhost";
+            Console.WriteLine("Service Host: {0}", host);
+
+            CallViaHttp();
+
+            CallViaNetTcp();
+        }
+
+        static void CallViaHttp()
+        {
+            var uri = string.Format("http://{0}/Service1.svc", host);
+            var address = new EndpointAddress(uri);
+            var binding = new BasicHttpBinding();
+            var factory = new ChannelFactory<IService1>(binding, address);
+            var channel = factory.CreateChannel();
+
+            Console.WriteLine(channel.Hello("WCF via Http"));
+        }
```

Comment:
You could output `"WCF via " + binding.Scheme` if you wanted to be fancy :)

Reply:
I would go for fancy normally 😄, but in this case, I prefer clarity/simplicity as it is a sample.